addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,53 +4,60 @@
-                            type="button"
-                            t-att-class="{
-                                'o-active': chatter.isComposerVisible and !chatter.isComposerLog,
-                            }"
-                            t-att-disabled="chatter.isDisabled"
-                            t-on-click="_onClickSendMessage">
-                        Send message
-                    </button>
-                    <button class="btn btn-link o_ChatterTopbar_buttonLogNote"
-                            type="button"
-                            t-att-class="{
-                                'o-active': chatter.isComposerVisible and chatter.isComposerLog,
-                            }"
-                            t-att-disabled="chatter.isDisabled"
-                            t-on-click="_onClickLogNote">
-                        Log note
-                    </button>
-                </t>
-                <t t-if="chatter.hasActivities">
-                    <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickScheduleActivity">
-                        <i class="fa fa-clock-o"/>
-                        Schedule activity
-                    </button>
-                </t>
-                <button class="btn btn-link o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickAttachments">
-                    <i class="fa fa-paperclip"/>
-                    <t t-if="chatter.isDisabled or chatter.thread and chatter.thread.areAttachmentsLoaded">
-                        <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                <div class="o_ChatterTopbar_actions">
+                    <t t-if="chatter.hasThread">
+                        <button class="btn btn-link o_ChatterTopbar_buttonSendMessage"
+                                type="button"
+                                t-att-class="{
+                                    'o-active': chatter.isComposerVisible and !chatter.isComposerLog,
+                                }"
+                                t-att-disabled="chatter.isDisabled"
+                                t-on-click="_onClickSendMessage">
+                            Send message
+                        </button>
+                        <button class="btn btn-link o_ChatterTopbar_buttonLogNote"
+                                type="button"
+                                t-att-class="{
+                                    'o-active': chatter.isComposerVisible and chatter.isComposerLog,
+                                }"
+                                t-att-disabled="chatter.isDisabled"
+                                t-on-click="_onClickLogNote">
+                            Log note
+                        </button>
```

Comment:
Because you have a diff on these lines:
make button ending `>` on new line with matching indent + indent attributes standard spacing (not aligned)